### ISSUE TITLE
build warning

### ISSUE THREAD

**Issue by mnot** [mnot/tt#1]
odd error:

~~~
mnot at Attitude-Adjuster in ~/Projects/tt
$ make -e ghpages
lib/main.mk:108: Forcing rebuild of .targets.mk
lib/main.mk:120: .targets.mk: No such file or directory
git fetch -q origin gh-pages:gh-pages
git clone -q -b gh-pages . /tmp/ghpages70992
date: illegal time format
usage: date [-jnRu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ... 
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]
~~~

It still builds, but I have no idea why this happens. @martinthomson?

**Comment by martinthomson** [comment]
Yeah, it's the mac/BSD thing again.  They are annoyingly different, though in this case I was using a feature that I perhaps should have expected to have fail.  Addressed in https://github.com/martinthomson/i-d-template/commit/fbbf48f931c4d7fd5f1e988511cd890e3743078e

**Comment by mnot** [comment]
Thx! Assuming the `.targets.mk` warning is normal?

**Comment by martinthomson** [comment]
Yeah, that's normal.  It's down to how I build special targets for getting different versions of the draft other than the current one. I can't suppress one of them and I haven't looked into the other, but it's harmless anyway.
